### PR TITLE
Develop++

### PR DESCRIPTION
**Pull request by manuelbarzi**

### CODE REVIEW

File: staff/manuelbarzi/js/challenges/scripts/toRandomCase.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+function toRandomCase(str) {
```
Comment (agandia9) [inline staff/manuelbarzi/js/challenges/scripts/toRandomCase.js:3]: use strict!

File: staff/manuelbarzi/js/challenges/scripts/toRandomCase.test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var input = 'Hello World';
```
Comment (agandia9) [inline staff/manuelbarzi/js/challenges/scripts/toRandomCase.test.js:3]: use strict!

File: stuff/js/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+
```
Comment (agandia9) [inline stuff/js/README.md:1]: use strict!

Comment (manuelbarzi) [reply stuff/js/README.md:1]: this is a readme, freunde! :)

**Pull request merged**